bbc/simorgh
# Updates to Indonesia, Japanese, Persian and Turkce navigation

Resolves JIRA n/a

Overall changes
======
Replaced the links to CPS Feature Indexes on the Indonesia and Japanese nav to the corresponging topics to avoid the redirect. 
Deleted the link to the Radio Feauture index in the Persian nav.
Added link to Local elections topic to the BBC Turkce nav

Code changes
======
- edited the navigation section of src/app/lib/config/services/indo.ts to add replace CPS Feature index links with links to topics
- edited the navigation section of src/app/lib/config/services/japanes.ts to add replace CPS Feature index links with links to topics
- edited the navigation section of src/app/lib/config/services/persian.ts to remove link to Radio CPS Feature index (last link)
- edited the navigation section of src/app/lib/config/services/turkce.ts to add link to /turkce/topics/cv2lyll8e02t
- Updated snapshots

Testing
======
1. Open BBC Indonesia front page the sixth and seven items in the nav should point to topic pages no CPS FIXs
2. Open BBC Japanese front page the third (ウクライナ侵攻), fourth (気候変動) and ninth (読み物・解説) item in the nav should point to topic pages no CPS FIXs
3. Open BBC Persian front page the last item in the nav should be Science (دانش) not Radio (رادیو)
4. Open BBC Turkce  front page the third item in the nav should be  31 Mart 2024 Yerel Seçimleri (/turkce/topics/cv2lyll8e02t)

======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
